src/Microsoft.ML.Data/DataView/SimpleRow.cs

```diff
@@ -70,7 +70,8 @@ public override bool IsColumnActive(int col)
         }
     }
 
-    public static class SimpleSchemaUtils
+    [BestFriend]
+    internal static class SimpleSchemaUtils
```

Comment:
>SimpleSchemaUtils [](start = 26, length = 17)

Now that creating a schema using the schema builder is much easier than it was back when this was much easier, what do we think about just deleting this outright and replacing the five usages?

Reply:
Sounds lovely!